src/app/CreateRecording/CreateRecording.tsx

```diff
@@ -65,9 +65,10 @@ const Comp: React.FunctionComponent< RouteComponentProps<{}, StaticContext, Crea
 
   const [activeTab, setActiveTab] = React.useState(0);
 
-  const handleCreateRecording = (recordingName: string, events: string, duration?: number): void => {
+  const handleCreateRecording = (recordingName: string, events: string, toDisk: boolean, 
```

Comment:
This function signature is getting pretty complicated with the number of arguments required in a specific order. Do you think it would make sense to refactor this to accept an object the same way that `api.createRecording` does, and simply pass that object through?

Reply:
Could I pass in a RecordingOptions object and keep the other values the same, or should I create a new object with everything needed?

Reply:
Could I pass in a RecordingOptions object and keep the other values the same, or should I create a new object with everything needed?

Reply:
IMO it makes slightly more sense to put all of the attributes into a single object, but I can see the reasoning behind putting the recording options into an object and keeping the name/duration/events split out. Up to you.

Reply:
I think for now its easier to use the recordingOptions object because we already have the interface for it, the problem is there is already an interface called Recording, we would have to create another one called newRecording or something to store everything together. I can change it to that if you want though!

Reply:
It could be named `RecordingAttributes`, or something. Don't name it `NewRecording`. Having another type with the same or similar name in a different context shouldn't be a strong consideration when deciding whether or how to structure or refactor things.

Reply:
Ok, yeah, its cleaner with a single object.